modules/user-access/pages/rbac-row-policy/row-policy-overview.adoc

```diff
@@ -0,0 +1,36 @@
+= RBAC Row Policy Overview
+
+Role-Based Access Control (RBAC) row policy is used to control access to specific rows (or for vertex level access control) of data in TigerGraph based on user roles and attribute values.
+
+==  User Guide
+The  row policy user guide has two parts:
+
+. xref:tigergraph-server:user-access:rbac-row-policy/rbac-row-policy.adoc[] - Learn the key concpets and features that make up row policy.
+. xref:tigergraph-server:user-access:rbac-row-policy/setup-row-policy.adoc[] - Learn how to setup a basic row policy using an example dataset.
+
+== xref:tigergraph-server:user-access:rbac-row-policy/row-policy-privlages-table.adoc[Row Policy Privileges Tables]
+Here you can find the row policy privileges tables for reference.
+
+== xref:tigergraph-server:user-access:rbac-row-policy/row-policy-ebnf.adoc[Row Policy EBNF]
+Here you can find the row policy EBNF examples for reference.
+
+== Row Policy Limitations
+* Because exception statements are not supported for `INTERPRET` mode, if a query is affected by a row policy, ensure to always install it before running it.
+
+* Currently, we only support row policy on vertices, not on edges.
+
+* A global vertex can only hold one global row policy and cannot apply a local row policy on a global vertex in a graph.
+
+* `LOADACCUM` will be blocked if the vertex type to be loaded has a row policy.
+
+* `SelectVertex` should always have a `vSet` assignment, if there are any row policies in the graph.
+
+* Statistics data will not be affected by row policies:
+** Blueprint function `outdegree()`
+** Commands that are related to built-in functions on graph, such as:
+*** `select count(*)` from `vertexType`
+*** xref:tigergraph-server:API:built-in-endpoints.adoc[].
+* Using the legacy privilege syntax, a user will receive a warning when trying to grant or revoke a privilege.
+* Row policy only support function privileges on the global scope.
```

Comment:
The the legacy privilege syntax for function privileges is only supported on the global scope.

Reply:
Thank you! I addressed this in my local branch and will commit the change.